15-FilteringEvents/UsingEventAggregator.Core/UsingEventAggregator.Core.csproj

```diff
@@ -38,12 +38,78 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <Reference Include="Prism, Version=7.1.0.431, Culture=neutral, PublicKeyToken=40ee6c3a2184dc59, processorArchitecture=MSIL">
-      <HintPath>..\packages\Prism.Core.7.1.0.431\lib\netstandard1.0\Prism.dll</HintPath>
-    </Reference>
+    <PackageReference Include="Microsoft.NETCore.Platforms">
```

Comment:
see comment for Core project on sample 12

Reply:
NU1202	Package Prism.Core 7.2.0.1367 is not compatible with portable45-net45+wp8 (.NETPortable,Version=v4.5,Profile=Profile49) / win10-arm64-aot. Package Prism.Core 7.2.0.1367 supports:
  - net45 (.NETFramework,Version=v4.5)
  - netcoreapp3.0 (.NETCoreApp,Version=v3.0)
  - netstandard2.0 (.NETStandard,Version=v2.0)	UsingCompositeCommands.Core	D:\Documents\source\repos\Prism-Samples-Wpf\12-UsingCompositeCommands\UsingCompositeCommands.Core\UsingCompositeCommands.Core.csproj

What can I do about this issue?

Reply:
again the entire csproj needs to be updated... the projects are currently old PCL's which wouldn't be compatible. You'll need to update them with the csproj contents I provided.

Reply:
> again the entire csproj needs to be updated... the projects are currently old PCL's which wouldn't be compatible. You'll need to update them with the csproj contents I provided.

I got it, removed all the old XML from .csproj file and replaced it with the one you have given.